### ISSUE TITLE
[Bug]: Ctrl+F shortcut doesn't do anything if search bar is visible on the screen

### ISSUE THREAD

**Issue by jezewskiBNetrox** [httpie/desktop#133]
### What happened?

Hi there,

not sure if it happens on Linux/Mac too but on WIndows with newest version of client there is a bit of problem with Ctrl+F shortcut.

When I press Ctrl+F on fresh response, search bar appears and I can search into response. But after I lose focus in search bar, Ctrl+F doesn't do anything. So I need to manually using mouse click on search bar to recover focus and search what I wanted. Very irritating. 

### Steps to reproduce?

```Markdown
1. Ask for any API to get a text/json response or even httpie.io and switch to the HTML view.
2. Press Ctrl+F.
3. Unfocus search bar.
4. Press Ctrl+F again.
5. Nothing happens.
```

### What did you expect to happen?

Recover of search bar focus to be able to change phrase I'm looking for in response.

### Platform

Windows

### HTTPie Desktop version

2022.3.6

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jezewskiBNetrox** [comment]
I can even reproduce this bug on Web version (https://httpie.io/app). I'm pretty sure it is not just in my case.

**Comment by jkbrzt** [comment]
✅ Fixed in [v2024.1.1 →](https://github.com/httpie/desktop/releases/tag/v2024.1.1)

**Comment by jezewskiBNetrox** [comment]
> ✅ Fixed in [v2024.1.1 →](https://github.com/httpie/desktop/releases/tag/v2024.1.1)

@jkbrzt Can confirm it works like it should now. Thank you very much!
